### PR TITLE
Minor tweaks to the 8.11 changelog.

### PR DESCRIPTION
**Pull request by Zimmi48**
In particular move one paragraph to put all the credits closer together.

### CODE REVIEW

File: doc/sphinx/changes.rst [hunk 1]
```diff
@@ -60,15 +60,15 @@ of |Coq| and affected releases. See the `Changes in 8.11+beta1`_
...
-Soegtrop, Théo Zimmermann worked on maintaining and improving the
-continuous integration system and package building infrastructure.
-
 Coq's documentation is available at https://coq.github.io/doc/v8.11/api (documentation of
 the ML API), https://coq.github.io/doc/v8.11/refman (reference
 manual), and https://coq.github.io/doc/v8.11/stdlib (documentation of
 the standard library).
 
+Maxime Dénès, Emilio Jesús Gallego Arias, Gaëtan Gilbert, Michael
+Soegtrop, Théo Zimmermann worked on maintaining and improving the
```
Comment (jfehrle) [inline doc/sphinx/changes.rst]:
```suggestion
Soegtrop and Théo Zimmermann worked on maintaining and improving the
```

Not quite grammatical without the "and".

File: doc/sphinx/changes.rst [hunk 2]
```diff
@@ -60,15 +60,15 @@ of |Coq| and affected releases. See the `Changes in 8.11+beta1`_
...
+continuous integration system and package building infrastructure.
+
 The OPAM repository for |Coq| packages has been maintained by
 Guillaume Claret, Karl Palmskog, Matthieu Sozeau, Enrico Tassi with
```
Comment (jfehrle) [inline doc/sphinx/changes.rst]:
```suggestion
Guillaume Claret, Karl Palmskog, Matthieu Sozeau and Enrico Tassi with
```

**Review by jfehrle — APPROVED** [review]
Looks good.  2 minor suggestions

**Comment by ppedrot** [comment]
Let's merge quickly.

**Pull request merged**